backside/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "lambda-golang-offline",
+    "description": "To run the golang lambda service locally",
+    "version": "1.0.0",
```

Comment:
Can we change the name to `sk8-backside`, remove the description and change the version to `0.0.1`, please?